### PR TITLE
Handle multiple exposure durations -  hazards/other hazards

### PR DESCRIPTION
**Pull request by sethdarragile6**
This PR populates the multiple exposures array of the Lighthouse submit/generatePDF request with hazard information coming in from the front end. Lighthouse then populates the 15E overflow of the 526ez digital form with each hazard and its corresponding date ranges.

## Summary

- *This work is behind a feature toggle (flipper): YES*
`disability_526_toxic_exposure`
- *(Summarize the changes that have been made to the platform)*
Modifications to the Form526ToLighthouseTransform logic + tests 
- *(If bug, how to reproduce)*
- *(What is the solution, why is this the solution?)*
- *(Which team do you work for, does your team own the maintenance of this component?)*
Disability Benefits Experience Team 1 (DBEX TREX 🦖)
- *(If introducing a flipper, what is the success criteria being targeted?)*

## Related issue(s)

- *Link to ticket created in va.gov-team repo OR screenshot of Jira ticket if your team uses Jira*
Handle multiple exposure durations - test through BE -15D #[83046](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/83046)
- *Link to previous change of the code/bug (if applicable)*
- *Link to epic if not included in ticket*

## Testing done

- [x] *New code is covered by unit tests*
- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
- *If this work is behind a flipper:*
  - *Tests need to be written for both the flipper on and flipper off scenarios. [Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample).*
  - *What is the testing plan for rolling out the feature?*

## Screenshots
image

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [x]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb [hunk 1]
```diff
@@ -53,8 +53,8 @@ class Form526ToLighthouseTransform # rubocop:disable Metrics/ClassLength
 
       HAZARDS = {
         asbestos: 'Asbestos',
-        chemical: 'SHAD (Shipboard Hazard and Defense)',
-        mos: 'Military Occupational Specialty (MOS)-related toxin',
+        chemical: 'SHAD - Shipboard Hazard and Defense',
+        mos: 'Military Occupational Specialty-related toxin',
```
Comment (freeheeling) [inline lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb:57]: Interesting that this hazard is not title-cased like all the others. Is that intentional?

Comment (freeheeling) [reply lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb:57]: Also, not part of this PR but does `other: 'OTHER'` in `HAZARDS_LH_ENUM` (L71) correspond to `none: 'None of these')` for `HAZARDS` (L61)?

**Review by freeheeling — APPROVED** [review]
Approved.

File: lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb (hunk 1 above)
Comment (sethdarragile6) [reply lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb]: > Interesting that this hazard is not title-cased like all the others. Is that intentional?
No, although it was signed off on this way 😂. I fixed it so it's at least consistent within this enum.

Comment (sethdarragile6) [reply lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb]: > Also, not part of this PR but does other: 'OTHER' in HAZARDS_LH_ENUM (L71) correspond to none: 'None of these') for HAZARDS (L61)?
No. Each of those options have their own specialized handling. `none` means the user checked 'None of these' and is used in this and gulf war + hazard enums as well. `other` is a specialized option for hazards which is set only when the freeform text field for "Other (specify):" has value, and is sent in order to check the OTHER box on the digital form

**Review by freeheeling — APPROVED** [review]
:shipit:

**Pull request merged**